README.md

```diff
@@ -24,7 +24,7 @@ There is a little bit of setup you need to do before testing the bots.
 
 ### hikari-lightbulb
 
-This library sports a very discord.py-like syntax (when making message commands), making for an easier time migrating for those wishing to switch. It is less powerful than hikari-tanjun, but is the more stable of the two options at present.
+This library sports a very discord.py-like syntax (when making message commands), making for an easier time migrating for those wishing to switch. It is less powerful than hikari-tanjun, but is the more stable of the three options at present.
```

Comment:
This would probably make more sense if we changed this to simply say it's the most stable, removing the Tanjun comparison.

Reply:
will do